developer/docs/help/context/keyboard-editor.md

```diff
@@ -213,17 +206,27 @@ Distinguish between left and right Ctrl/Alt
 :   If checked, treats the left and right Ctrl/Alt as separate layers
 
 Display 102nd Key (as on European keyboards)
-:   European keyboards have one extra key that is not on US keyboards.
+:   European keyboards (ISO layout) have one extra key that is not on US keyboards (ANSI layout).
     This key is positioned to the right of the left shift key. Some
     other keyboards have additional keys; these are not shown on the
-    layout designer. If you have a European layout selected as your
+    layout designer. 
+    
+    If you have a European layout selected as your
     Windows layout, the 102nd key will always be visible in the
     designer; ensure you select this checkbox if you want the 102nd key
     to always be visible to end users of the layout, irrespective of
-    their selected base layout. When a European layout is selected as
+    their selected base layout. 
+    
+    When a European layout is selected as
     your Windows layout, the shape of the Enter key will also change to
     take two rows, and the backslash key will move down one row, but not
-    otherwise change in behaviour.
+    otherwise change in behaviour. 
+
+    <img src="../images/ui/OnScreenKeyboard_ISO.png" style="width:100%" alt="Keyboard Editor - On-Screen tab with ISO layout"/>
+    *See the difference between the two layouts in Keyman Developer.*
+
+    > [!NOTE]
+    > To ensure compatibility with both ANSI and ISO physical keyboards, you should tick the "Display 102nd Key (as on European keyboards)" checkbox in Keyman Developer.
```

Comment:
I'm not sure this really is accurate. The additional key can be left unassigned without harm. In fact, for greatest compatibility for all users, you should _not_ assign this key, because users on ANSI keyboards will not be able to access it!